coreneuron/CMakeLists.txt

```diff
@@ -193,6 +199,20 @@ add_custom_command(TARGET coreneuron
                    COMMENT "Running nrnivmodl-core with halfgap.mod")
 add_dependencies(coreneuron scopmath)
 
+# =============================================================================
+# create nrnivmodl-core
+# =============================================================================
+# coreneuron executable
+include_directories(${CORENEURON_PROJECT_SOURCE_DIR})
+add_executable(nrniv-core "apps/coreneuron.cpp")
+target_link_libraries(nrniv-core
+        corenrnmech
+        ${MPI_C_LIBRARIES}
+        ${MPI_CXX_LIBRARIES}
```

Comment:
I believe MPI_CXX_LIBRARIES could be removed

Reply:
True, I copy-pasted that without thinking too much :)